src/storage/btree.h

```diff
@@ -41,6 +41,7 @@
 #include "query_evaluator.h"
 #include "lock_manager.h"
 #include "recovery.h"
+#include "thread.h"
```

Comment:
Please remove all added #include "thread.h" that are not necessary.  I am already removing them in the thread patch and it will only conflict with my work.

Reply:
did it ... but there is a hidden dependency there: THREAD_ENTRY is used but the declaration is not included directly